.github/CODEOWNERS

```diff
@@ -39,6 +39,7 @@ exporter/signalfxcorrelationexporter/      @open-telemetry/collector-contrib-app
 exporter/splunkhecexporter/                @open-telemetry/collector-contrib-approvers @atoulme
 exporter/stackdriverexporter/              @open-telemetry/collector-contrib-approvers @dashpole
 exporter/sumologicexporter/                @open-telemetry/collector-contrib-approvers @pmm-sumo @sumo-drosiek
+exporter/uptraceexporter/                  @open-telemetry/collector-contrib-approvers @vmihailenco
```

Comment:
Could you double-check that dependabot also includes this module?

Reply:
@jpkrohling I don't remember adding it but it seems to be there already - https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/dependabot.yml#L105-L108